styles/settings.scss

```diff
@@ -0,0 +1,11 @@
+/* #TODO: Get this to work */
+$theme-show-notifications: 'false';
+/* GENERAL */
+$theme-image-path: '/img';
+
+/* TYPOGRAPHY */
+
```

Comment:
I think the font path is missing from here, I am seeing build errors on NextJS

Reply:
✅